### ISSUE TITLE
TypeError: Cannot read property 'server_sample_rate' of null

### ISSUE THREAD

**Issue by taymonbeal** [gwillen/solstice-audio-test#26]
```
TypeError: Cannot read property 'server_sample_rate' of null
    at query_server_clock (https://echo.jefftk.com/net.js:242:45)
    at async ServerConnection.start (https://echo.jefftk.com/net.js:51:31)
    at async try_increase_batch_size_and_reload (https://echo.jefftk.com/app.js:1449:5)
    at async MessagePort.handle_message (https://echo.jefftk.com/app.js:1196:7)
```

**Comment by taymonbeal** [comment]
This looks like it might be the server sending a bad response. I'd like to see a network log from when this is triggered.

**Comment by taymonbeal** [comment]
Fixed in 569c6a62eeb831ccd0c7d1c31b83b92be174ef5c.
